### PR TITLE
refactor: update with respect to documentation

### PR DESCRIPTION
**Pull request by MoskalykA**
## Closes [#15](https://github.com/deepgram-devs/deepgram-deno-sdk/issues/15)

### CODE REVIEW

File: types/transcription/wordBase.ts [hunk 1]
```diff
@@ -3,8 +3,4 @@ export type WordBase = {
   start: number;
   end: number;
   confidence: number;
-  punctuated_word?: string;
```
Comment (MichaelJolley) [inline types/transcription/wordBase.ts:6]: These properties do exist on the word. They should remain.

File: types/transcription/prerecordedTranscriptionOptions.ts [hunk 2]
```diff
@@ -1,55 +1,60 @@
...
    * @see https://developers.deepgram.com/api-reference/speech-recognition-api#operation/transcribeAudio/properties/version
    */
   version?: string;
+
   /**
    * BCP-47 language tag that hints at the primary spoken language.
    * @default en-US
-   * @remarks Possible values are en-GB, en-IN, en-NZ, en-US, es, fr, ko, pt,
-   * pt-BR, ru, tr or null
+   * @remarks Possible values are zh-CN, zh-TW, nl, en, en-AU, en-IN, en-NZ, en-GB, en-US, nl, fr, fr-CA, de, hi, hi-Latn, id, it, ja,
```
Comment (MichaelJolley) [inline types/transcription/prerecordedTranscriptionOptions.ts:34]: We add languages so often we probably want to just let them see the documentation for the most up to date list. Let's remove this remark.

**Review by MichaelJolley — CHANGES_REQUESTED** [review]
@MoskalykA thanks for this. There are a few changes needed before we can merge.

These feature additions should also affect some the `usageOptions` type and usage responses on some requests to the `/usage` endpoints.

**Comment by MoskalykA** [comment]
Good 👍🏻

**Review by briancbarrow — APPROVED** [review]

**Pull request merged**